src/c++/perf_analyzer/genai-perf/tests/test_llm_metrics.py

```diff
@@ -215,34 +221,34 @@ def test_triton_llm_profile_data(self, prepare_triton_profile_data) -> None:
 
         assert stat.avg_time_to_first_token == 2
         assert stat.avg_inter_token_latency == 2.25
-        avg_rott = 31 / ns_to_sec(63)
-        assert stat.avg_output_token_throughput_per_request == pytest.approx(avg_rott)
+        avg_ottpr = 31 / ns_to_sec(63)
```

Comment:
`rott` to `ottpr`: Changed to avoid confusion in acronym. `rott` was used to abbreviate "request output token throughput" but we changed that to "output token throughput per request".